components/member-profile/contribution/index.js

```diff
@@ -41,8 +41,14 @@ const Contribution = ({ contribution, fullName, imageLink, devUser }) => {
     }
   }
 
+  let url = featureUrl || prList[0]?.url;
```

Comment:
why not use `const` for `url` rather than `let`? We are not re-assigning the value.

Reply:
yes that's right changing it